anesthetic/samples.py

```diff
@@ -148,6 +150,10 @@ def plot(self, ax, paramname_x, paramname_y=None, *args, **kwargs):
                     xmin=xmin, xmax=xmax, ymin=ymin, ymax=ymax,
                     *args, **kwargs)
 
+    docstrings.delete_params('plot.parameters',
+                             'ax', 'paramname_x', 'paramname_y')
+
+    @docstrings.dedent
```

Comment:
Instead of deleting parameters from the docstring there is also the option of specifying which ones to keep. Not sure what in this case would seem more appropriate.
1) Keep only `plot_type` meaning that when parameters are added to `plot` they _won't_ automatically be documented in `plot_1d`.
2) Delete `ax`, `paramname_x`, and `paramname_y` meaning that new parameters to `plot` _will_ show up in the documentation of `plot_1d` unless removed.

Reply:
The first case is definitely safer -- It's always better for documentation to be absent than wrong.